app/models/poll.rb

```diff
@@ -1,4 +1,13 @@
 class Poll < ActiveRecord::Base
+  include HasPublicAuthor
+  include Flaggable
+  acts_as_votable
```

Comment:
A Poll is not votable, we use a different voting system for Polls.

Reply:
Is that method not intended for votes in comments? Placed it there because in other places that uses comments has it.

Reply:
The Comment model has already acts_as_votable, that makes comments votable, but this is on the Poll model, the Poll is not votable in the same way as comments, acts_as_votable is a gem that acts on the model you put it not over comments or other related entities

Reply:
Right. I understand.